Libraries/MiKTeX/Core/Session/init.cpp

```diff
@@ -952,7 +954,16 @@ void SessionImpl::SetEnvironmentVariables()
     }
   }
   MIKTEX_ASSERT(!gsDirectories.Empty());
+
+#if MIKTEX_WINDOWS
```

Comment:
Should be: `#if defined(MIKTEX_WINDOWS)`

Reply:
I will change this.

Reply:
The `#if MIKTEX_WINDOWS` above (line 925) also requires a change. But not as part of this change set.